From c34c9e672d8a1d24df42c79cfe06426c2d8dad94 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Sat, 12 Feb 2011 23:19:53 +0000
Subject: [PATCH] gitignore: add test-mktemp to ignore list
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change the .gitignore to ignore test-mktemp which is built from
test-mktemp.c. Arnout Engelen added this in
6cf6bb3e47ac2f667fa0b27a4222e903ff6fb77c but did not add a
corresponding entry to .gitignore.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 .gitignore |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/.gitignore b/.gitignore
index 3dd6ef7..c460c66 100644
--- a/.gitignore
+++ b/.gitignore
@@ -170,6 +170,7 @@
 /test-index-version
 /test-line-buffer
 /test-match-trees
+/test-mktemp
 /test-obj-pool
 /test-parse-options
 /test-path-utils
-- 
1.7.2.3

